File: URI_CPP_UFRR/1044.cpp
#include <iostream>
int main(){
	int a,b,temp;
	std::cin >> a >> b;
	temp = 0;
	if (a > b){
		temp = b;
	    b = a;
	    a = temp;	
	}	    
	if((b%a)==0){std::cout<<"Sao Multiplos\n";}
	else{std::cout<<"Nao sao Multiplos\n";}
	return 0;	
}

File: URI_Python_UFRR/1044.py
linha1 = input().split(' ')
a, b = map(int, linha1)
temp = 0
if a > b:
    temp = b
    b = a
    a = temp
    
if (b%a)==0:print('Sao Multiplos')
else:print('Nao sao Multiplos')
    